calcom/cal.com
# fix: metadata for seats in webhooks

## What does this PR do?

Add metadata with videoCallUrl to webhooks payload for a seat where a booking already exists (+ fix workflow MEETING_URL variable for seats)

Fixes #13712

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

- Create seated event type with Cal Video as location
- Create 'Booking created' webhook
- Book a slot
- Book same slot again 
  - Check payload of triggered webhook and see that metadata with videoCallUrl is here
